### DISCUSSION TITLE
maximum devices connected simultaneously

### DISCUSSION THREAD

**Discussion by ffreitasb** [dchristl/esp32_nat_router_extended#121] [Q&A] [answered]
What is the maximum number of devices I can connect simultaneously? I have a NAT with ESP8266 that can handle a maximum of 8 due to hardware limitations, what is the maximum of the ESP32?

**Comment by dchristl** [comment, marked as answer]
Hello, 

the  limitation on the ESP32 is 10 devices.

**Reply by ffreitasb** [reply]
Thank you, dchristl!
